Cargo.toml

```diff
@@ -9,6 +9,8 @@ members = [
   "oak_client",
   "oak_containers_stage1",
   "oak_core",
+  "oak_containers_example_trusted_application",
+  "oak_containers_example_untrusted_application",
```

Comment:
I also think we could shorted `application` to just `app`, because it's already a very long name

Reply:
sure, why not